cmd/sequencer/helpers.go

```diff
@@ -0,0 +1,70 @@
+package sequencer
+
+import (
+	"fmt"
+
+	"github.com/astria/astria-cli-go/internal/keys"
+	"github.com/astria/astria-cli-go/internal/sequencer"
+	"github.com/pterm/pterm"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// GetPrivateKeyFromFlags retrieves the private key from the command flags.
+// If the 'privkey' flag is set, it returns the value of that flag.
+// If the 'keyring-address' flag is set, it calls the 'PrivateKeyFromKeyringAddress' function
+// to retrieve the private key from the keyring.
+// If the 'keyfile' flag is set, it calls the 'PrivateKeyFromKeyfile' function
+// to retrieve the private key from the keyfile.
+// If none of the flags are set or if the value of 'keyfile' is empty, it returns an error.
+// NOTE - this requires the flags `keyfile`, `keyring-address`, and `privkey`
+func GetPrivateKeyFromFlags(c *cobra.Command) (string, error) {
```

Comment:
This can be called with all three flags which results in privkey (least secure one) to return. Should either default to Keyfile instead in such cases or require only one flag to be set in parent commands.

Reply:
Good catch. I'll change the order or priority for this func, but also add `MarkMutuallyExclusive` on the command to ensure only one flag can be used at a time.